airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -225,6 +225,7 @@ class KubernetesPodOperator(BaseOperator):
         container name to use.
     :param deferrable: Run operator in the deferrable mode.
     :param poll_interval: Polling period in seconds to check for the status. Used only in deferrable mode.
+    :param log_pod_spec_on_failure: If pod specification should be logged in a case of task fail.
```

Comment:
```suggestion
    :param log_pod_spec_on_failure: Log the pod's specification if a failure occurs
```

Let's follow the lead of `log_events_on_failure`